packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1716,6 +1716,32 @@ export class Vector3 {
         return Scalar.NormalizeRadians(angle);
     }
 
+    /**
+     * Gets the angles between two points needed to face the target from the start
+     * @param start the starting point
+     * @param target the target point
+     * @param ref the vector3 to store the result
+     * @returns the updated ref with the angles between the two points in the form (pitch, yaw, roll)
+     */
+    public static PitchYawRollToMoveBetweenPointsToRef(start: Vector3, target: Vector3, ref: Vector3): Vector3 {
+        const diff = TmpVectors.Vector3[0];
+        target.subtractToRef(start, diff);
+        ref.y = Math.atan2(diff.x, diff.z) || 0;
+        ref.x = Math.atan2(Math.sqrt(diff.x ** 2 + diff.z ** 2), diff.y) || 0;
+        return ref;
+    }
+
+    /**
+     * Gets the angles between two points needed to face the target from the start

```

Comment:
Since the method return a Vector3 this cannot be desribed as angles. Change comments to

```
 /**
     * Gets the rotation that aligns the roll axis (Y) to the line joining the start point to the target point
     * @param start the starting point
     * @param target the target point
     * @returns the rotation in the form (pitch, yaw, 0)
     */
```